### ISSUE TITLE
Bandit command fails

### ISSUE THREAD

**Issue by dave-kennedy** [bittner/pyclean#30]
I'm not familiar with bandit. It looks like it works but then it throws an error:

```
$ tox -e bandit
GLOB sdist-make: /Users/dave.kennedy/Code/pyclean/setup.py
bandit inst-nodeps: /Users/dave.kennedy/Code/pyclean/.tox/.tmp/package/1/pyclean-2.0.0.zip
bandit installed: bandit==1.7.0,gitdb==4.0.5,GitPython==3.1.14,pbr==5.5.1,pyclean @ file:///Users/dave.kennedy/Code/pyclean/.tox/.tmp/package/1/pyclean-2.0.0.zip,PyYAML==5.4.1,six==1.15.0,smmap==3.0.5,stevedore==3.3.0
bandit run-test-pre: PYTHONHASHSEED='3226865627'
bandit run-test: commands[0] | bandit --ini tox.ini -r pyclean setup
[main]	INFO	Using command line arg for excluded paths
[main]	INFO	Using command line arg for selected targets
[main]	INFO	Using command line arg for recursive scan
[main]	INFO	Using command line arg for aggregate output type
[main]	INFO	Using command line arg for max code lines output for issue
[main]	INFO	Using command line arg for severity level
[main]	INFO	Using command line arg for confidence level
[main]	INFO	Using command line arg for output format
[main]	INFO	Using command line arg for output file
[main]	INFO	profile include tests: None
[main]	INFO	profile exclude tests: None
[main]	INFO	cli include tests: None
[main]	INFO	cli exclude tests: None
[main]	INFO	running on Python 3.9.1
Run started:2021-03-05 00:34:05.581256

Test results:
>> Issue: [B110:try_except_pass] Try, Except, Pass detected.
   Severity: Low   Confidence: High
   Location: pyclean/py3clean.py:131
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b110_try_except_pass.html
130	                rmdir(directory)
131	            except Exception:
132	                pass
133	

--------------------------------------------------
>> Issue: [B404:blacklist] Consider possible security implications associated with subprocess module.
   Severity: Low   Confidence: High
   Location: pyclean/pypyclean.py:16
   More Info: https://bandit.readthedocs.io/en/latest/blacklists/blacklist_imports.html#b404-import-subprocess
15	import shutil
16	import subprocess
17	import sys

--------------------------------------------------
>> Issue: [B603:subprocess_without_shell_equals_true] subprocess call - check for execution of untrusted input.
   Severity: Low   Confidence: High
   Location: pyclean/pypyclean.py:33
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b603_subprocess_without_shell_equals_true.html
32	    """Iterate through all python modules in an installed Debian package"""
33	    p = subprocess.Popen(('dpkg', '-L', package), stdout=subprocess.PIPE,
34	                         stderr=subprocess.PIPE)
35	    files, stderr = p.communicate()

--------------------------------------------------
>> Issue: [B603:subprocess_without_shell_equals_true] subprocess call - check for execution of untrusted input.
   Severity: Low   Confidence: High
   Location: pyclean/pypyclean.py:96
   More Info: https://bandit.readthedocs.io/en/latest/plugins/b603_subprocess_without_shell_equals_true.html
95	    """
96	    p = subprocess.Popen(('dpkg', '-L', package), stdout=subprocess.PIPE,
97	                         stderr=subprocess.PIPE)
98	    stdout, stderr = p.communicate()

--------------------------------------------------

Code scanned:
	Total lines of code: 549
	Total lines skipped (#nosec): 0

Run metrics:
	Total issues (by severity):
		Undefined: 0.0
		Low: 4.0
		Medium: 0.0
		High: 0.0
	Total issues (by confidence):
		Undefined: 0.0
		Low: 0.0
		Medium: 0.0
		High: 4.0
Files skipped (1):
	setup (No such file or directory)
ERROR: InvocationError for command /Users/dave.kennedy/Code/pyclean/.tox/bandit/bin/bandit --ini tox.ini -r pyclean setup (exited with code 1)
________________________________________________ summary ________________________________________________
ERROR:   bandit: commands failed
```

**Comment by bittner** [comment]
[Bandit](https://pypi.org/project/bandit/) is a security issue scanner for Python code.

If I see this correctly all complaints come from legacy code. We're not going to fix this. Hence, simply ignore all those cases.

Maybe it would make sense to maintain a [baseline](https://pypi.org/project/bandit/#baseline) to make it easier to ignore those – irrelevant – issues. But TBH I've really discovered this feature just now.

P.S.: The output above shows a bug in the Tox configuration. I fixed that with commit 196d42f. Thanks for bringing that up! :medal_sports: 
```console
setup (No such file or directory)
```

**Comment by dave-kennedy** [comment]
Yeah, the setup error is what I was referring to. Thanks for fixing!
